app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md

```diff
@@ -504,8 +504,6 @@ keywords:
 
 If you’ve worked as an unqualified teacher, you may be able to get [qualified teacher status (QTS)](/what-is-qts) through an assessment only programme.
 
-If you already have QTS, find out how you can [return to teaching](/returning-to-teaching).
```

Comment:
Should we keep this link and point to TV @philspray @EllieNodder?

Reply:
I was too keen with the delete key!